kustomize/preprod/simple-app/deployment.yaml

```diff
@@ -13,6 +17,32 @@ spec:
       labels:
         app: simple-app
     spec:
+      volumes:
+        - name: env-volume
+          emptyDir: {}
+      initContainers:
+        - name: env-creator
+          image: alpine
+          command:
+            [
+              "/bin/sh",
+              "-c",
+              "echo MY_SECRET=$(echo $MY_SECRET) > /env/.env && echo NAME=$(echo $NAME) >> /env/.env",
+            ]
+          env:
+            - name: MY_SECRET
```

Comment:
It will be annoying also to add our envs here, cannot we try to add them all all at once ?

```yaml
          envFrom:
            - configMapRef:
                name: my-configs
            - secretRef:
                name: my-secrets
                
```

Reply:
Updated! Please re-review and send me feedbacks. Thanks!